examples/nginx/site-available

```diff
@@ -0,0 +1,52 @@
+# This is an example nginx `sites-available` file to use with maxwell-daemon.
+# Lua scripting is used to communicate with the daemon.
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    location / {
+        # Arbitrate the request's path (defaulting to master).
+        set $maxwell "master";
+        access_by_lua '
```

Comment:
Since this is derived from the code we're already using, I'm just going to add a comment about that.